controller/customdomain_utils.go

```diff
@@ -142,13 +142,16 @@ func (r *CustomDomainReconciler) returnIngressToClusterIngressOperator(reqLogger
 
 	delete(customIngress.Labels, managedLabelName)
 	customIngress.Spec.NodePlacement = &operatorv1.NodePlacement{
-		NodeSelector: &metav1.LabelSelector{
-			MatchLabels: map[string]string{"node-role.kubernetes.io/worker": ""},
+		Tolerations: []corev1.Toleration{
+			{
+				Key:      "node-role.kubernetes.io/infra",
+				Effect:   corev1.TaintEffectNoSchedule,
+				Operator: corev1.TolerationOpExists,
+			},
```

Comment:
I believe, you also need to tolerate `node-role.kubernetes.io/worker` as well.

Reply:
Are you sure? I didn't think that worker nodes had any taints? Otherwise, Cx workloads would not be able to schedule on them. Also - the previous `nodeSelector` configuration wouldn't have worked.

Reply:
Good point, yes no taints for workers.